templates/_catalog/frontendframeworks.json

```diff
@@ -26,5 +26,19 @@
       "version": "7.2.0",
       "preview": true
     }
+  },
+  {
+    "name": "Vue",
+    "displayName": "Vue.js",
+    "summary": "Vue.js is an open source, incrementally adoptable, front-end framework created by Evan You for building interactive web interfaces.",
```

Comment:
Defines the short description seen in tiles. Actually I need help with a long description if anyone can provide one that would be great (seen when you click details. Currently there is no long description.